core/src/main/scala/kafka/utils/LogDirUtils.scala

```diff
@@ -18,15 +18,16 @@
 package kafka.utils
 
 import kafka.controller.LogDirEventNotificationHandler
+import kafka.zk.KafkaZkClient
 import scala.collection.Map
 
 object LogDirUtils extends Logging {
 
   private val LogDirEventNotificationPrefix = "log_dir_event_"
   val LogDirFailureEvent = 1
 
-  def propagateLogDirEvent(zkUtils: ZkUtils, brokerId: Int) {
-    val logDirEventNotificationPath: String = zkUtils.createSequentialPersistentPath(
+  def propagateLogDirEvent(zkClient: KafkaZkClient, brokerId: Int) {
+    val logDirEventNotificationPath: String = zkClient.createSequentialPersistentPath(
```

Comment:
we need to add this method  to KafkaZkClient. This is similar imilar to KafkaZkClient.createAclChangeNotification method. 
Related zk nodes/paths are : ZkData.LogDirEventNotificationZNode, ZkData.LogDirEventNotificationSequenceZNode